From: Mans Rullgard <mans@mansr.com>
Date: Mon, 13 Oct 2008 19:32:16 +0000 (+0100)
Subject: ARM: Add prompt for CONFIG_ALIGNMENT_TRAP
X-Git-Url: http://git.mansr.com/?p=linux-omap;a=commitdiff_plain;h=60d60f0ca47fcf4fbb649e45aa64f5a0a4c2f2c8

ARM: Add prompt for CONFIG_ALIGNMENT_TRAP

This adds a prompt text for CONFIG_ALIGNMENT_TRAP, thus making it
visible in make *config.

Signed-off-by: Mans Rullgard <mans@mansr.com>
---

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 61314f6..18d3119 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -971,7 +971,7 @@ config LEDS_CPU
 	  will overrule the CPU usage LED.
 
 config ALIGNMENT_TRAP
-	bool
+	bool "Enable alignment trap"
 	depends on CPU_CP15_MMU
 	default y if !ARCH_EBSA110
 	help
